templates/common/head-links.html

```
@@ -3,12 +3,12 @@
 		<link rel="icon" type="image/png" sizes="32x32" href="{% static "favicons/favicon-32x32.png" %}">
 		<link rel="icon" type="image/png" sizes="16x16" href="{% static "favicons/favicon-16x16.png" %}">
 		<link rel="manifest" href="{% static "favicons/site.webmanifest" %}">
-		<link rel="mask-icon" href="{% static "favicons/safari-pinned-tab.svg" %}" color="#ec1c24">
+		<link rel="mask-icon" href="{% static "favicons/safari-pinned-tab.svg" %}" color="var(--cinemata-red-500)">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)